UnitySDK/Assets/ML-Agents/Scripts/SideChannel/FloatPropertiesChannel.cs

```diff
@@ -0,0 +1,125 @@
+using System.Collections.Generic;
+using System.IO;
+using System;
+using System.Text;
+
+namespace MLAgents
+{
+
+    public interface IFloatProperties
+    {
+        /// <summary>
+        /// Sets one of the float properties of the environment. This data will be sent to Python.
+        /// </summary>
+        /// <param name="key"> The string identifier of the property.</param>
+        /// <param name="value"> The float value of the property.</param>
+        void SetProperty(string key, float value);
+
+        /// <summary>
+        /// Get an Environment property with a default value. If there is a value for this property,
+        /// it will be returned, otherwise, the default value will be returned.
+        /// </summary>
+        /// <param name="key"> The string identifier of the property.</param>
+        /// <param name="defaultValue"> The default value of the property.</param>
+        /// <returns></returns>
+        float GetPropertyWithDefault(string key, float defaultValue = 0f);
```

Comment:
nit: do we need the default for `defaultValue`? `GetPropertyWithDefault("myProperty")` sounds weird